tests/helpers.rs

```diff
@@ -181,19 +125,22 @@ pub async fn wait_for_server(url: &str) {
 /// stdout
 pub async fn run_wash_command<I, S>(args: I) -> Vec<u8>
 where
-    I: IntoIterator<Item = S>,
+    I: IntoIterator<Item = S> + std::clone::Clone,
     S: AsRef<std::ffi::OsStr>,
 {
     let output = Command::new("wash")
-        .args(args)
+        .args(args.clone())
         .output()
         .await
         .expect("Unable to run wash command");
     if !output.status.success() {
         panic!(
-            "wash command didn't exit successfully: {}",
-            String::from_utf8_lossy(&output.stderr)
-        )
+            "wash command ({:?}) didn't exit successfully: {}",
+            args.into_iter()
+                .map(|i| i.as_ref().to_str().unwrap_or_default().to_owned())
+                .collect::<Vec<String>>(),
+            String::from_utf8_lossy(&output.stderr),
+        );
```

Comment:
I'm also wondering if this should actually return a `Result` instead so that you could localize the error where it happens, rather than awkwardly getting a reference to the `panic!(...)` when it fails.